README.md

```diff
@@ -27,6 +27,7 @@ The Bamboo.MandrillAdapter and Bamboo.SendgridAdapter **are being used in produc
 and have had no issues. It's also pretty simple to [create your own adapter]. Feel free
 to open an issue or a PR if you'd like to add a new adapter to the list.
 
+* `Bamboo.MailgunAdapter` - Ships with Bamboo. Thanks to @princemaple.
```

Comment:
> If someone did a large feature almost entirely by themselves, then I like to do it. It’s a nice sign and I think that it will encourage more contribution. I love it when I contribute and the maintainers mention it :)

👍